biocypher_metta/adapters/gene_ontology_adapter.py

```diff
@@ -0,0 +1,34 @@
+from biocypher_metta.adapters.ontologies_adapter import OntologyAdapter
+
+class GeneOntologyAdapter(OntologyAdapter):
+    def __init__(self, write_properties, add_provenance, ontology, type, label="ontology term", dry_run=False):
+        super().__init__(write_properties, add_provenance, ontology, type, label, dry_run)
```

Comment:
Minor but better to pass arguments to the parent class after instantiating the fields of the current class